addon/helpers/includes.js

```diff
@@ -10,7 +11,7 @@ export function includes(needleOrNeedles, haystack) {
   let needles = isEmberArray(needleOrNeedles) ? needleOrNeedles : [needleOrNeedles];
   let haystackAsEmberArray = emberArray(haystack);
```

Comment:
```suggestion
  let haystackAsEmberArray = emberArray(asArray(haystack));
```
This would possibly fix the issue with frozen arrays and the `{{includes}}` helper. I created a fiddle for reproduction: https://ember-twiddle.com/94cbfbbea6e38e60d1966b19fbd9dc60